web/src/lib/nostr/pool.ts

```diff
@@ -231,6 +245,46 @@ export class Pool {
         })
     }
 
+    public async sendReaction(message, reaction: string) {
+        const noteId = message.id;
+        const notePubkey = message.pubkey;
+
+        console.log('******* SENDING REACTION: ' + reaction + ' to note with Id: ' + noteId + ' pubkey: ' + notePubkey);
+
+        if (reaction.length !== 1) {
+            console.error('Trying to send reactions with > 1 character is not allowed by nip-25');
+            return;
+        }
+
+        if (!noteId || !notePubkey) {
+            return false;
+        }
+
+        // 1
+        const event: Event = {
+            kind: 7,
+            content: reaction,
+            created_at: Math.floor(Date.now() / 1000),
```

Comment:
This calculation also appeared quite a few times in your previous commits. Maybe have it in utils.ts?

Reply:
In the next PR I've changed it to 0, and then I fill it just once before publishing.